src/test/java/org/alliancegenome/curation_api/ExpressionBulkUploadFmsITCase.java

```diff
@@ -66,6 +67,19 @@ public void init() {
 	public void expressionBulkUploadAllFields() throws Exception {
 		loadRequiredEntities();
 		checkSuccessfulBulkLoad(expressionBulkPostEndpoint, expressionTestFilePath + "AF_01_all_fields.json");
+
+		RestAssured.given().when()
+			.header("Content-Type", "application/json")
+			.body("{}")
+			.post(experimentFindEndpoint)
+			.then()
+			.statusCode(200)
+			.body("totalResults", is(1))
+			.body("results", hasSize(1))
+			.body("results[0].dataProvider.sourceOrganization.abbreviation", is("ZFIN"))
+			.body("results[0].uniqueId", is(experimentUniqueIdExpected))
```

Comment:
Should be checking that all fields of ExpressionExperiment are set and returned as expected.

Reply:
Checks added